.github/workflows/build-and-test.yaml

```diff
@@ -41,20 +41,23 @@ jobs:
         uses: arduino/setup-protoc@v1
         with:
           version: '3.17.3'
-      - name: Test and Build Feathub Python
+      - name: Build and test Feathub Python library
         run: |
           python -m pip install --upgrade pip setuptools wheel
 
           protoc -I=python/feathub/common/protobuf/ --python_out=python/feathub/common/protobuf/ value.proto
           
-          python -m pip install ./python
           python -m pip install -r python/dev-requirements.txt
+          python -m pip install ./python
           
+          # Re-install black here because its pip dependency conflicts with
+          # apache-flink 1.15.2.
+          python -m pip install black==22.3.0
           python -m black --check python
```

Comment:
nit: use `./python` instead of `python` here.

Reply:
It is fixed now.